### PR TITLE
[editor] Fix closed status

### PR DESCRIPTION
**Pull request by mgsergio**
@Zverik

### CODE REVIEW

File: 3party/opening_hours/rules_evaluation_private.hpp [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 namespace osmoh
 {
-bool IsActive(Timespan const & spsn, std::tm const & date);
+bool IsActive(Timespan spsn, std::tm const & date);
```
Comment (Zverik) [inline 3party/opening_hours/rules_evaluation_private.hpp]: spsn или span?

Comment (mgsergio) [inline 3party/opening_hours/rules_evaluation_private.hpp]: span, thanks

Comment (mgsergio) [inline 3party/opening_hours/rules_evaluation_private.hpp]: Done.

**Comment by Zverik** [comment]
LGTM

**Comment by Zverik** [comment]
DUDE JTTP

**Pull request merged**